src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import racingcar.util.RandomNumberGenerator;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void move(RandomNumberGenerator randomNumberGenerator) {
```

Comment:
List에 담긴 모든 car를 옮길 때 사용하는 것 같은데, 메소드 명을 더 명확히 해주면 좋을 것 같습니다!
```suggestion
    public void moveAll(RandomNumberGenerator randomNumberGenerator) {
```